README.md

```diff
@@ -23,8 +22,38 @@ ember install ember-title
 Usage
 ------------------------------------------------------------------------------
 
-[Longer description of how to use the addon in apps.]
+Applications can choose to set their title in their templates (via the
+`{{title}}` helper), or in routes (by setting a `title` property). These titles
+are gathered from both the route and template and combined into the final title
+to be displayed.
+
+Basic features are:
+
+* Combining title from each "layer"
+* Specify custom separator (e.g. `Foo | Bar` vs `Foo > Bar`)
+* Provide a replacement instead of concatenating
+* Specify the ordering (leaf -> root, or root -> leaf)
+
 
+### Route Based Title
+
+In your route files you can set the `title` property to a string:
+
+```
+export default class extends Route {
+  title = "My Awesome Page";
+}
+```
+
+### Template Based Title
+
+In your template (any template!) you can invoke the `{{title}}` helper:
```

Comment:
If there are 2 hbs files that have invoked the `{{title}}` helper, would that:
1. concatenate both the title strings along with the routes `title` property or
2. replace the entire `<title>` HTML element with the string that was passed?